src/dmd/dsymbolsem.d

```diff
@@ -1551,13 +1551,6 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
 
         if (dsym.type.toBasetype().ty == Terror)
             dsym.errors = true;
-
-        if(sc.scopesym && !sc.scopesym.isAggregateDeclaration())
-        {
-            for (ScopeDsymbol sym = sc.scopesym; sym && dsym.endlinnum == 0;
-                 sym = sym.parent ? sym.parent.isScopeDsymbol() : null)
-                dsym.endlinnum = sym.endlinnum;
```

Comment:
This is valid usage of the member necessary for debug information.

Reply:
can you show me where this is used, I've  browsed files and it doedsn't seem to be used. by cv.d and alikes.

Reply:
It's here: https://github.com/dlang/dmd/blob/master/src/dmd/tocsym.d#L319

Reply:
tocsym was not modified by this PR, looks like it's another linnum maybe

Reply:
dsym is of type VarDeclaration, not DScopeSymbol.

Reply:
I see, it's used but indirectly, after copy.

Reply:
Maybe that a an offset from loc.linnum could be stored in a ushort instead of the full value